server/apiRouter.js

```diff
@@ -50,4 +52,6 @@ router.post('/transaction-line-items', transactionLineItems);
 router.post('/initiate-privileged', initiatePrivileged);
 router.post('/transition-privileged', transitionPrivileged);
 
+router.post('/auth/create-with-idp', createWithIdp);
```

Comment:
I find this server route quite confusing. What are we creating? Could the name be more self-explanatory: `create-user-with-idp`?

(and then related variable names should be changed accordingly)

Reply:
I.e. `/auth/auth-with-idp` is kind of understandable, but `/auth/create-with-idp` makes you wonder what's happening.

Reply:
The corresponding Flex API endpoint is `/current_user/create_with_idp`. There the `/current_user` path gives context to the "create" part. So how about `/auth/create-user-with-idp`?

Reply:
@lyyder yeah, that's what I was after in the first comment. I just forgot the "/auth/" part.